transit_odp/frontend/assets/sass/components/forms/_forms.scss

```diff
@@ -11,28 +11,34 @@
 }
 
 .seasonal-service {
-  .licence {
-    margin-bottom: 5rem;
-  }
+    .licence {
+        margin-bottom: 5rem;
+    }
 
-  .buttons {
-    margin-top: 5rem;
-  }
+    .buttons {
+        margin-top: 5rem;
 
-  .govuk-heading-s {
-    margin-top: govuk-spacing(6);
-    margin-bottom: govuk-spacing(6);
-  }
+        button {
```

Comment:
so you have just added a button!!

Reply:
I reduced the gap between buttons if you look in dev there is a gap that is not in the design